ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -0,0 +1,142 @@
+SectionVerb {
```

Comment:
Remove empty blocks, please.

Reply:
it was a placeholder for actual register settings for the es8336. I am still not clear on which ones are needed.

Reply:
done